client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -29,10 +30,18 @@ import Header from './components/header';
 import { name, settings } from './onboarding-block';
 import { Slot as SidebarSlot } from './components/sidebar';
 import SettingsSidebar from './components/settings-sidebar';
+import { State as OnboardingState } from './stores/onboard/reducer';
+import { STORE_KEY } from './stores/onboard';
+import { Steps } from './types';
 import './stores/domain-suggestions';
-import './stores/onboard';
 import './stores/verticals-templates';
 import './style.scss';
+import '../../../apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss';
+
+const stepCompleted: Record< Steps, ( state: OnboardingState ) => boolean > = {
+	[ Steps.IntentGathering ]: ( { siteTitle } ) => !! siteTitle,
```

Comment:
Site title is optional so we should let the user continue just with the vertical selected.
```suggestion
	[ Steps.IntentGathering ]: ( { siteVertical } ) => !! siteVertical,
```

Reply:
Oh, because of the "I don't know yet" option? I wonder if that really makes sense, UX-wise. After all, a lot of things depend on this, e.g. the default subdomain :thinking:

Reply:
Yes, there are some UX decisions to be made ( cc: @dwolfe ) but let's keep following the idea of getting the user to FSE as quickly as possible with the possibility of (not mandatory) customisation.

Reply:
Okay, I'll apply your suggestion :+1: